File: app/src/main/java/com/ikvych/cocktail/data/network/DrinkApiService.java
package com.ikvych.cocktail.data.network;

import com.ikvych.cocktail.data.entity.DrinkApiResponse;

import retrofit2.Call;
import retrofit2.http.GET;
import retrofit2.http.Query;

public interface DrinkApiService {

    @GET("api/json/v1/1/search.php")
    Call<DrinkApiResponse> getDrinksByName(@Query("s") String drinkName);
}

File: app/src/main/java/com/ikvych/cocktail/data/network/DrinkApiService.kt
package com.ikvych.cocktail.data.network

import com.ikvych.cocktail.data.entity.DrinkApiResponse
import com.ikvych.cocktail.data.entity.IngredientApiResponse
import retrofit2.Call
import retrofit2.http.GET
import retrofit2.http.Query

interface DrinkApiService {

    @GET("api/json/v1/1/search.php")
    fun getDrinksByName(@Query("s") drinkName: String?): Call<DrinkApiResponse?>

    @GET("api/json/v1/1/list.php?i=list")
    fun getAllIngredients(): Call<IngredientApiResponse?>
}